packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/components/ZoomingScatterPlot/ZoomingScatterPlot.js

```diff
@@ -205,55 +207,31 @@ function ZoomingScatterPlot({
   return (
     <VisXZoom
       constrain={constrain}
-      data={data}
       height={height}
-      invertAxes={invertAxes}
       left={leftPosition}
-      margin={margin}
-      padding={padding}
       panning={panning}
-      parentHeight={parentHeight}
-      parentWidth={parentWidth}
-      tickDirection={tickDirection}
       top={topPosition}
       width={width}
-      zoomingComponent={ScatterPlot}
+      zoomingComponent={(zoomProps) => (
+        <ScatterPlot {...zoomProps} {...scatterPlotProps} />
+      )}
       zoomConfiguration={zoomConfiguration}
       zooming={zooming}
-      {...props}
```

Comment:
I think `setOnPan` and `setOnZoom` that are included in the old `...props` / or new `...scatterPlotProps` need to be passed as props to the `<VisXZoom>` component here for the toolbar to work.

Reply:
Good catch. The list of props for `ScatterPlotViewer` is quite long and confusing. 

https://github.com/zooniverse/front-end-monorepo/blob/aed398d41e9aa30d9690e726e47ecde13f99c30f/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/components/ScatterPlot/ScatterPlot.js#L224-L286

Reply:
Fixed!

Reply:
Weird that this didn't break the tests at all (but did break the zoom in/out buttons for scatter plot stories in the storybook.)